src/main/scala/uk/gov/nationalarchives/notifications/messages/EventMessages.scala

```diff
@@ -275,6 +335,8 @@ object EventMessages {
     }
 
     override def sqs(incomingEvent: GenericMessagesEvent, context: Unit): Option[SqsMessageDetails] = Option.empty
+
+    override def sns(incomingEvent: GenericMessagesEvent, context: Unit): Option[SnsMessageDetails] = Option.empty
```

Comment:
This is one of the sns overrides you can get rid of if you provide a default in the trait.